### ISSUE TITLE
SAFT Achilles heel with the SEC

### ISSUE THREAD

**Issue by vincek39** [saft-project/saft#7]
The SAFT model is an ambitious, creative and convoluted attempt to get ICOs in line with SEC regulations.

IMO the weak link is that assumption that newly issued tokens can pass the Howey test. Investors will continue to have a) expectation of profits b) from efforts of others.

Some thoughts from my recent twitter thread at https://twitter.com/VinceKuraitis/status/914996753279066112

1/ Problem w/ SAFT model is assuming that value creation occurs by creating technology. In fact, value creation occurs with network adoption
2/ Does creation/existence of unadopted software give investors any better insight into future network adoption? Doubtful.
3/ Further re: SAFT, greatest ICO value creation will occur ONLY AFTER network adoption reaches critical mass & NETWORK EFFECT kicks in.
4/ Even after tokens are widely available to public, investors will have a) expectations of profits b) from efforts of others (Howey #fail)
5) Likely implication: SEC will continue to view #ICO tokens issued under SAFT model as "securities"
6/ Think about it. The term growth "hacking" describes level of experimentation & effort required to grow a network. Doesn't just happen.
7/ More on this line of thinking re: SAFT at https://www.linkedin.com/pulse/why-utility-tokens-can-still-securities-even-issued-only-newman-nahas/

**Comment by SMickelsen** [comment]
I agree that plenty of the AFUT value is created due to post issuance efforts of the developers, although in my view the AFUT is still a commodity.  The CFTC considers Bitcoin to be a commodity and stated so in the Coinflip/Derivabit order 2 years ago.  The SEC noted the CFTC's classification in its order disproving the listing of the Winkelvoss Bitcoin Trust and analyzed the trust as a commodity trust with the CFTC as the primary regulator for the bitcoin spot market.  Other tokens are significantly different than Bitcoin, but the analytical structure holds.
  
On the securities/commodity question the whitepaper hits it correctly, although it underestimates the CFTC regulatory oversight of AFUT transactions. Every AFUT transaction will be subject to CFTC jurisdiction and regulations.

**Comment by Luckydee** [comment]
Sometimes i wonder why CTFC and SEC duplicate there role when it comes to ICO,
